fake-timers.js

```diff
@@ -1726,7 +1726,10 @@ function withGlobal(_global) {
             timer.func.apply(null, timer.args);
         } else {
             /* eslint no-eval: "off" */
-            eval(timer.func);
```

Comment:
We should probably make another PR to make sure the node behavior is more correct.

Reply:
So this PR is ok for now? 
And I need to find out how this package "knows" on which env it is, because I'm aware that on Node it needs to **not** be possible at all. Originally overall there should be 3 PRs.

Reply:
Yes, this PR is ok, I will give other members a few days to look at it and if all looks good we'll land it

Reply:
By the way, thank you for helping all the time. Your help was, and is, needed! :)